exporter/zipkinexporter/zipkin.go

```diff
@@ -169,43 +197,41 @@ func (ze *zipkinExporter) Start(host exporter.Host) error {
 }
 
 func (ze *zipkinExporter) Shutdown() error {
```

Comment:
The `Shutdown` method is not really needed anymore, delete it and also delete `Name` and `Start`. Because of the deletion of `Shutdown` you will need to delete the usage of the shutdown option too, `exporterhelper.WithShutdown(ze.Shutdown)`